Core/src/main/java/org/tribuo/transform/transformations/MeanStdDevTransformation.java

```diff
@@ -160,13 +162,25 @@ public void observeValue(double value) {
         }
 
         @Override
-        public void observeSparse() { }
+        @Deprecated
+        public void observeSparse() {
+            observeValue(0.0);
+        }
 
         @Override
-        public void observeSparse(int count) { }
+        public void observeSparse(int sparseCount) {
```

Comment:
Why is this copied here and in `BinningTransformer`?

Reply:
Do you mean I should refactor out the update logic? That would involve introducing some class to carry the multiple returns. The reason it's in both is because binning transformer can convert things into bins and this one standardises the data, so it's still real valued in this case and implicitly categorical in the binning transformer case.

Reply:
that's fair enough. I wasn't clear on the purpose of the two implementations, but if one is rescaling and the other is binning then it's probably simple enough (and the logic is simple enough) to copy the logic.